core/domain/exp_services.py

```diff
@@ -237,6 +237,42 @@ def get_multiple_explorations_by_id(exp_ids, strict=True):
     return result
 
 
+def get_displayable_exploration_summaries_matching_ids(exploration_ids,
+    user_id):
+    """Given a list of exploration ids, filters the list for 
+    explorations that are currently non-private and not deleted, 
+    and returns a list of dicts of the corresponding exploration summaries.
+    """
+    displayable_exploration_summaries = []
+    exploration_summaries = get_exploration_summaries_matching_ids(
+        exploration_ids)
+
+    for exploration_summary in exploration_summaries:
+        if exploration_summary and exploration_summary.status != (
+            rights_manager.ACTIVITY_STATUS_PRIVATE):
```

Comment:
indent this by 4 more, and deindent lines 253 to 271 by 4.

Reasoning: lines 253 to 271 are the 'next' scope and should be indented only one level deeper, not two. To distinguish line 252 from the others, we indent it by an additional 4.